<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectPropsDialog</class>
 <widget class="QDialog" name="ObjectPropsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>240</height>
   </size>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Object Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>78</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>78</height>
      </size>
     </property>
     <property name="title">
      <string>Add</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushAddPropertyButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Add New</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="errorLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>17</height>
         </size>
        </property>
        <property name="font">
         <font>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { color : red; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <item>
      <property name="text">
       <string>sadasd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>asdasdsad</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dasdsadsa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dasdasd</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
